packages/sample/src/main.ts

```diff
@@ -20,4 +21,45 @@ program
         await led();
     });
 
+program
+    .command("analog <port> [continuous]")
+    .description(
+        "Read the analog value of the given port. Specify 'true' as the " +
+            "second argument to run continuously.",
```

Comment:
Let's use a `--continuous` flag instead on all of these.

Reply:
Converted to flag